Added full support for game banners, GUI enhancements & more

LegItMate opened the pull request:
This PR adds support for banners of all the games from launchers that we support as of right now.

The GUI enhancements can be seen here
![DS9Z3y5tlP](https://user-images.githubusercontent.com/77503777/148994479-f66538ab-9982-4513-9c00-053d7c72fad5.gif)

Known issues:
- The pfp resets to default after every restart
- Steam Games throws error on not finding games

**It will not work for people who do not have access to the organization, since it requires the self-hosted API made for Lazap**

DashCruft commented:
Please update your source code and fork with the latest update in Nightly branch to resolve more then half of the conflicts that are happening in this pr if possible to speed up the checking process.

Thank you

LegItMate commented:
Okay! I have fixed all the conflicts and also added some more GUI changes. Feel free to test it and give me feedback(s) on it or merge it if you think it is good :)

DashCruft commented:
Great work!

Merging to Nightly now...
